ersteops/emergency/migrations/0015_auto_20220517_1937.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 2.2.27 on 2022-05-17 19:37
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emergency', '0014_emergency_sales_rep'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='attentionderivation',
+            name='emergency',
+            field=models.ForeignKey(default=1, on_delete=django.db.models.deletion.DO_NOTHING, related_name='derivation_emergency_name', to='emergency.Emergency', verbose_name='emergencia'),
+        ),
+        migrations.AlterField(
+            model_name='attentionderivation',
+            name='hospital',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.DO_NOTHING, related_name='attention_hospital_name', to='emergency.AttentionHospital', verbose_name='hospital'),
+        ),
+        migrations.AlterField(
+            model_name='emergency',
+            name='attention_final_grade',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='final_attention_kind_name', to='emergency.AttentionKind', verbose_name='Grado de atención final'),
+        ),
+        migrations.AlterField(
+            model_name='emergency',
+            name='grade_type',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.DO_NOTHING, related_name='attention_kind_name', to='emergency.AttentionKind', verbose_name='Grado Emergencia'),
+        ),
+        migrations.AlterField(
+            model_name='emergency',
+            name='has_paid',
+            field=models.BooleanField(blank=True, default=False, verbose_name='Estatus de Pago'),
+        ),
+        migrations.AlterField(
+            model_name='emergency',
+            name='service_category',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.DO_NOTHING, related_name='service_category_name', to='emergency.ServiceCategory', verbose_name='Tipo de emergencia'),
+        ),
+        migrations.AlterField(
+            model_name='emergency',
+            name='zone',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.DO_NOTHING, related_name='zone_name', to='emergency.AttentionZone', verbose_name='Zona de Atención'),
+        ),
+        migrations.AlterField(
+            model_name='emergencyderivation',
+            name='hospital',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.DO_NOTHING, related_name='em_attention_hospital_name', to='emergency.AttentionHospital', verbose_name='hospital'),
```

Comment:
Cuando se borre alguna llave foránea, o lo protegemos o le ponemos en cascada nO?
O cuál es la razón de dejarlo en DO_NOTHING? @vezino